File: EQIS_env-main/react_sample/react/src/interfaces/lang/langListGrid.ts
import { cloneDeep } from 'lodash';

export interface IlangListGrid {
    mlgCd: string;
    langCd: string;
    mlgSbc: string;
}

export const langListGridInit = () => {
    const langListGrid: IlangListGrid = {
        mlgCd: '',
        langCd: '',
        mlgSbc: '',
    };

    return cloneDeep(langListGrid);
};

File: EQIS_env-main/react_sample/springboot/Spring/src/main/java/com/sample/basic/sample/code/model/LangListGrid.java
package com.sample.basic.sample.code.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@Data
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class LangListGrid {
	private String mlgCd;
	private String langCd;
	private String mlgSbc;
}
